lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -857,4 +860,225 @@ void SessionStateEnforcer::set_pdr_attributes(
           .redirect_server_address());
 }
 
+void SessionStateEnforcer::aggregate_records(

```

Comment:
All the functions here are copy&paste from our current code. Reuse of the code is a must.
This will have to be refactored to handle reporting of 5g 

https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Auri200%2Fmagma.git&branch=feg_merge_credit_reports_into_single_crru&file=lte%2Fgateway%2Fc%2Fsession_manager%2FLocalEnforcer.cpp&editor=JetBrains&version=v1.2.1&utm_product_name=CLion&utm_product_version=2021.1.2&start_row=375&start_col=25&end_row=375&end_col=25

Reply:
Yes @uri200 almost all functions are similar. 
LocalEnforcer object is used for storing for 4g sessions.
SessionStateEnforcer object is used for storing for 5g sessions.

We can not use local defined functions of 4g for 5g. Will not get 5g session info.

Reply:
please elavorate with `Will not get 5g session info.` 

Sessiond should be able to know the type of record without us copying&pasting code